tests/unit/code/test_files.py

```diff
@@ -14,6 +17,15 @@ def test_files_fix_ignores_unsupported_extensions():
     assert not files.apply(path)
 
 
+def test_files_fix_ignores_unsupported_language():
+    languages = create_autospec(Languages)
+    files = Files(languages)
+    # pylint: disable=protected-access
+    files._extensions[".py"] = None
+    path = Path('unsupported.py')
```

Comment:
```suggestion
    path = Path('unsupported.unsupported')
```

Reply:
That would fall under unsupported extension (tested line 12), not unsupported language

Reply:
(that said not sure it makes sense to support extensions without a language, but that's a separate conversation)